### ISSUE TITLE
ST3 build 3083

### ISSUE THREAD

**Issue by justrjlewis** [facelessuser/BracketHighlighter#242]
I recently updated ST to build 3083 and my BH has been acting as if it was "reset" to default.  I double checked my user/bh_core file (I keep a backup) and it's correct, but when I open ST3 wherever the cursor is initially is what is highlighted according to my settings and it just stays frozen that way and gives a white underline wherever my cursor is.

screenshots: 
http://cloud.justrjlewis.com/image/262t311G1g05
http://cloud.justrjlewis.com/image/423P371P283V

I have not changed/ done anything other than update ST.  HELP!

**Comment by facelessuser** [comment]
Are you sure BH has been updated to the latest as well?

See https://github.com/facelessuser/BracketHighlighter/issues/226.

I don't know what ST version you are on, but around build 3070 some things changed that required me to make some changes in BH to keep  Sublime from freezing.  Jon of Sublime reached out to me, and in order to prevent sublime freezing up if when people upgraded Sublime without upgrading BH, we implemented a check to disable BH if the BH version didn't have the fix needed.
